airflow/operators/check_operator.py

```diff
@@ -248,14 +275,21 @@ def execute(self, context=None):
         ratios = {}
         test_results = {}
         for m in self.metrics_sorted:
-            if current[m] == 0 or reference[m] == 0:
-                ratio = None
+            cur = current[m]
+            ref = reference[m]
+            threshold = self.metrics_thresholds[m]
+            if cur == 0 or ref == 0:
+                ratios[m] = None
+                test_results[m] = self.ignore_zero
             else:
-                ratio = float(max(current[m], reference[m])) / \
-                    min(current[m], reference[m])
-            self.log.info("Ratio for %s: %s \n Ratio threshold : %s", m, ratio, self.metrics_thresholds[m])
-            ratios[m] = ratio
-            test_results[m] = ratio < self.metrics_thresholds[m]
+                ratios[m] = self.ratio_formulas[self.ratio_formula](current[m], reference[m])
+                test_results[m] = ratios[m] < threshold
+
+            self.log.info("Current metric for %s: %s", m, cur)
+            self.log.info("Past metric for %s: %s", m, ref)
```

Comment:
do we need that many logging info lines? Let's change it to debug. WDYT?

Reply:
I feel like we only turn on debug mode when there is an Airflow internal issue and we are trying to debug the Airflow internal issue. 

For this case, task failure is usually caused by incorrect user behavior, e.g. wrong SQL (not related to Airflow internal), and we should provide the information users need to debug their issues.

Reply:
Ok, then could we make it a single log line(we could add \d in between) which is same as previous case?

Reply:
fixed.